lldb/include/lldb/Expression/DiagnosticManager.h

```diff
@@ -130,13 +120,17 @@ class DiagnosticManager {
       m_diagnostics.back()->AppendMessage(str);
   }
 
-  // Returns a string containing errors in this format:
-  //
-  // "error: error text\n
-  // warning: warning text\n
-  // remark text\n"
+  /// Returns a string containing errors in this format:
+  ///
+  ///     "error: error text\n
+  ///     warning: warning text\n
+  ///     remark text\n"
+  LLVM_DEPRECATED("Use GetAsStatus instead", "GetAsStatus()")
```

Comment:
Why does this need to be marked deprecated ? It's internal API, we could just get rid of it, right ?

Reply:
I'll remove that, that was just for development.

Reply:
Done!